src/Entity/Dataset.php

```diff
@@ -388,22 +388,6 @@ public function getAuthors()
         }
     }
 
-    /**
-     * Get reference date shortcut, to get Authors from DatasetSubmission.
-     *
-     * @return string|null
-     */
-    public function getReferenceDateYear()
```

Comment:
Removing this now may break other things until other tickets are merged.

Reply:
Yes, this pull request can be merged at the end